### PR TITLE
fixed "bancodesio.it" as in #83128

### PR DESCRIPTION
**Pull request by albertopasqualetto**
Refer to issue #83128

### CODE REVIEW

**Comment by krystian3w** [comment]
IMO this allow load first-party tracking, maybe better use something like here said: https://github.com/AdguardTeam/AdguardFilters/issues/83128#issuecomment-840559240

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 1]
```diff
@@ -1575,3 +1575,4 @@ lg.com#@#.cookie-check
 mycard2go.com#@#.cookiebar
 fashionette.de#@##cookies-notice
 foodnetwork.co.uk#@#.cookie-message:not(html):not(body)
+@@||bancodesio.it^$domain=bancodesio.it
```
Comment (krystian3w) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:1578]:
```suggestion
`@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/$~third-party`
```

or:

```diff
--- /eu_cookie_$script,stylesheet
+++ /eu_cookie_$script,stylesheet,domain=~bancodesio.it
```

in  `/AnnoyancesFilter/sections/cookies_general.txt`?

**Review by krystian3w — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 2]
```diff
@@ -1575,4 +1575,4 @@ lg.com#@#.cookie-check
 mycard2go.com#@#.cookiebar
 fashionette.de#@##cookies-notice
 foodnetwork.co.uk#@#.cookie-message:not(html):not(body)
-@@||bancodesio.it^$domain=bancodesio.it
+`@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/$~third-party
```
Comment (krystian3w) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:1578]: Oops a missed remove 1 of 2 "`" from suggestion form.

```suggestion
@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/$~third-party
```

**Review by krystian3w — CHANGES_REQUESTED** [review]

**Review by krystian3w — APPROVED** [review]

**Comment by krystian3w** [comment]
Maybe no fix issue on iOS/iPadOS env: https://github.com/AdguardTeam/AdguardFilters/issues/83128#issuecomment-840568002

The option to support content blocker technology is to remove `$~third-party` or change into `domain=bancodesio.it`; as PR has been blocked due to conflicting modifications on main branch, this can be left for the AdGuard Team to consider which fix would be more effective.

**Comment by albertopasqualetto** [comment]
Ok, I don't know what is better, in any case I also read your comment in the issue and we are talking about a (small) bank website, so there are not so much problems in allowing a bit more contents.

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 3]
```diff
@@ -1575,3 +1575,4 @@ lg.com#@#.cookie-check
 mycard2go.com#@#.cookiebar
 fashionette.de#@##cookies-notice
 foodnetwork.co.uk#@#.cookie-message:not(html):not(body)
+@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/$~third-party
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:1578]: `@@||bancodesio.it/sites/all/modules/eu_cookie_compliance/js/jquery.cookie-` is enough. `~third-party` is redundant in this case.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 3 above)
Comment (albertopasqualetto) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:1578]: ok

**Review by zloyden — APPROVED** [review]

**Pull request merged**